docs/animations.md

```diff
@@ -0,0 +1,80 @@
+# Animations
+
+Animations currently reside in `packages/animation`. The top level logic is held in `packages/animation/src/components/provider.js`. This package is responsible for taking a serializable array of animations objects and transforming the objects into WAAPI & AMP animations as well as exposing aggregate animation methods: `pause`, `play`, etc
+
+**Note:** Editing and updating the serializable array of animation objects is considered part of the story editor and held on the page level in the story reducer state.
+
+## Generating an Animation from a Serializable Animation Object
+
+Our serializable animation objects currently have this structure:
+
+```js
+// sample animation object
+{
+  id, // unique identifier for this animation
+  type, // type of animation, `blink`, `bounce` etc.
+  targets, // what elements in the story this animation is targeting
+  ...args // whatever args this type of animation takes ie `ease` | `duration`
```

Comment:
I think the linter's struggling w/ the `...args` here

Reply:
same. updated it. hopefully fine now